lib/govuk_index/presenters/expanded_links_presenter.rb

```diff
@@ -32,6 +32,10 @@ def taxons
       content_ids("taxons")
     end
 
+    def facet_groups
+      content_ids("facet_groups")
```

Comment:
At the moment, the link from facet to facet group is in the `parent` links array https://github.com/alphagov/govuk-content-schemas/blob/master/formats/facet.jsonnet#L90 
I'm not sure where we get `facet_groups` from in expanded links

Reply:
Having discussed this we've agreed that content either needs `facet_groups` links or the facet group to be present in the `facet_values` links, the former is more explicit see: https://github.com/alphagov/govuk-content-schemas/commit/90be6c695ae3f5827790c68d10cee7b61f16c7ae